crates/store/re_types/tests/types/line_strips2d.rs

```diff
@@ -1,35 +1,44 @@
 use re_types::{
-    archetypes::LineStrips2D,
-    components::{ClassId, Color, DrawOrder, LineStrip2D, Radius},
-    Archetype as _, AsComponents as _,
+    archetypes::LineStrips2D, components, Archetype as _, AsComponents as _, ComponentBatch,
 };
 
 #[test]
 fn roundtrip() {
     let expected = LineStrips2D {
         #[rustfmt::skip]
         strips: vec![
-            LineStrip2D::from_iter([[0., 0.], [2., 1.], [4., -1.], [6., 0.]]), //
-            LineStrip2D::from_iter([[0., 3.], [1., 4.], [2., 2.], [3., 4.], [4., 2.], [5., 4.], [6., 3.]]), //
-        ],
-        radii: Some(vec![
-            Radius::from(42.0), //
-            Radius::from(43.0),
-        ]),
-        colors: Some(vec![
-            Color::from_unmultiplied_rgba(0xAA, 0x00, 0x00, 0xCC), //
-            Color::from_unmultiplied_rgba(0x00, 0xBB, 0x00, 0xDD),
-        ]),
-        labels: Some(vec![
-            "hello".into(),  //
-            "friend".into(), //
-        ]),
-        draw_order: Some(DrawOrder(300.0.into())),
-        class_ids: Some(vec![
-            ClassId::from(126), //
-            ClassId::from(127), //
-        ]),
-        show_labels: Some(false.into()),
+            components::LineStrip2D::from_iter([[0., 0.], [2., 1.], [4., -1.], [6., 0.]]), //
+            components::LineStrip2D::from_iter([[0., 3.], [1., 4.], [2., 2.], [3., 4.], [4., 2.], [5., 4.], [6., 3.]]), //
+        ]
+        .serialized()
+        .map(|batch| batch.with_descriptor_override(LineStrips2D::descriptor_strips())),
```

Comment:
Ah right, we need to do this manually in the tests now… oh well

Reply:
Well, we only need to do it for that specific test because that's the thing I'm testing for. In any other case I sure hope people will use the builder methods!